scripts/create-dnsmasq.sh

```diff
@@ -2,6 +2,7 @@
 basedir=".."
 outputdir="output/dnsmasq"
 path="${basedir}/cache_domains.json"
+hosts_path="/etc/dnsmasq.hosts"
```

Comment:
Looking at this in more detail, I feel this should by default be respective to ${outputdir}, i'd suggest possibly changing the script to do something like:

outputdir="output"
lancacheconf=${outputdir}/etc/dnsmasq.d
hosts_path=/etc/dnsmasq.hosts
destfilepath=${outputdir}/hosts_path (And appropriately update to using ${destfilepath}/${destfilename} when writing to in the script)

Then if you wanted to run this in situ you would set outputdir to "" and it would run on the base directory, else it'll store everything in outputdir ready for copying across

Reply:
I agree with this and have made the necessary changes (will push later).